analyzers/rspec/cs/S3871_c#.html

```diff
@@ -18,8 +18,7 @@ <h2>Compliant Solution</h2>
 </pre>
 <h2>Exceptions</h2>
 <p>This rule ignores Exception types that are not derived directly from <code>System.Exception</code>, <code>System.SystemException</code>, or
-<code>System.ApplicationException</code>.</p>
-<h2>See</h2>
+<code>System.ApplicationException</code>. == See</p>
```

Comment:
please fix this before next release @cristian-ambrosini-sonarsource @csaba-sagi-sonarsource 

https://peach.sonarsource.com/coding_rules?languages=cs&q=S3871&open=csharpsquid%3AS3871

![image](https://user-images.githubusercontent.com/38876598/200324597-f3f2b9a7-2122-4920-bf3d-2d05bcf0a218.png)

Reply:
PR to fix it: https://github.com/SonarSource/sonar-dotnet/pull/6310